apps/next-product-starter/src/styles/globals.css

```diff
@@ -1,5 +1,6 @@
 @import './icons.css';
 @import "~@skillrecordings/player/dist/skillrecordings-player.css";
+@import "~@skillrecordings/quiz/dist/styles.css";
```

Comment:
I noticed that if this would be imported in `pages/_app.tsx` it would automatically take precedence over all tailwind classes (if present).